lib/App/GHPT/WorkSubmitter.pm

```diff
@@ -17,6 +17,13 @@ use WebService::PivotalTracker 0.10;
 
 with 'MooseX::Getopt::Dashes';
 
+has create_story => (
+    is  => 'ro',
+    isa => Bool,
+    documentation =>
+        'If true, will create a new story instead of finding an existing one.',
+);
+
 has project => (
     is  => 'ro',
     isa => Str,
```

Comment:
A coworker suggests putting a `default => $ENV{APP_GHPT_PROJECT},` in to `project`'s definition. Is that something you would be okay with?

Reply:
Yes, that makes sense.